prac_05/hex_colours.py

```diff
@@ -0,0 +1,28 @@
+"""
+CP1404/CP5632 Practical
```

Comment:
Good docstring :)
However are you doing both CP1404 and CP5632?
I would change it to (your subject) Practical

Reply:
I kept the CP5 stuff because it was in the provided examples.